src/audio/detect_test.c

```diff
@@ -613,7 +613,7 @@ static int test_keyword_copy(struct comp_dev *dev)
 
 	/* copy and perform detection */
 	cd->detect_func(dev, source,
-			source->avail / comp_frame_bytes(source->source));
+			source->avail / KPB_NR_OF_CHANNELS);
```

Comment:
@mrajwa better to change it to comp_frame_bytes(dev) to make it more understandable.

Reply:
yes, good suggestion.

Reply:
@mrajwa Please change it and we can merge after full CI is passed.

Reply:
Done. This fix has already been merged into release branch.